api/config/routes.rb

```diff
@@ -91,14 +91,7 @@
     resources :states, only: [:index, :show]
 
     resources :taxonomies do
-      member do
-        get :jstree
-      end
-      resources :taxons do
-        member do
-          get :jstree
```

Comment:
I'm more a fan of killing it, as it wasn't ever actually published in the public API on the old docs site:
https://guides.spreecommerce.com/api/taxonomies.html

We can make a note in the release notes pointing to the commit if they want to pull that code into their own stores. I would consider that a better upgrade path than supporting it as an extension.

Though not removing the routes and just axing the javascript is still a good improvement... Maybe we do leave it till we just deprecate the entirety of the API.